tests/infra/clients.py

```diff
@@ -327,31 +327,18 @@ def request(self, request):
             self.stream.update(rc.stdout)
         return request.id
 
-    # TODO: https://github.com/microsoft/CCF/issues/612
     def signed_request(self, request):
-        raise NotImplementedError(
-            "Signed requests are not yet implemented with curl. Make sure to unset CURL_CLIENT envvar."
-        )
-
         with tempfile.NamedTemporaryFile() as nf:
             msg = getattr(request, "to_{}".format(self.format))()
             LOG.debug("Going to send {}".format(msg))
             nf.write(msg)
             nf.flush()
-            dgst = subprocess.run(
-                ["openssl", "dgst", "-sha256", "-sign", self.key, nf.name],
-                check=True,
-                capture_output=True,
-            )
-            subprocess.run(["cat", nf.name], check=True)
             cmd = [
-                "curl",
-                "-v",
+                "./scurl.sh",
+                "--fail",
```

Comment:
Why do we want curl to fail silently?

Reply:
As discussed, now replaced with `-w` option to display status code in last line of output.